content/en/docs/concepts/services-networking/ingress-controllers.md

```diff
@@ -51,7 +51,7 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 * The [Kong Ingress Controller for Kubernetes](https://github.com/Kong/kubernetes-ingress-controller#readme)
   is an ingress controller driving [Kong Gateway](https://konghq.com/kong/).
 * [Kusk Gateway](https://kusk.kubeshop.io/) is an OpenAPI-driven ingress controller based on [Envoy](https://www.envoyproxy.io).
-* The [NGINX Ingress Controller for Kubernetes](https://www.nginx.com/products/nginx-ingress-controller/)
+* The [NGINX Ingress Controller for Kubernetes](https://docs.nginx.com/nginx-ingress-controller/)
   works with the [NGINX](https://www.nginx.com/resources/glossary/nginx/) webserver (as a proxy).
```

Comment:
Link https://www.nginx.com/resources/glossary/nginx/ also returns 403.
Please also modify this link.

Reply:
It's also OK to drop the second link